lib/tasks/sidekiq/cleanup.rake

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+namespace :sidekiq do
+  desc 'cleanup BID-* keys from sidekiq-batch'
```

Comment:
```suggestion
  desc 'cleanup BID-* keys older than 3 hours from sidekiq-batch'
```

Reply:
actually... maybe we can pass this as an argument, so it can be decided in the runtime, whether it's 3 hours, or 1 day, or whatever else

Reply:
ideally - yes